ts/main.ts

```diff
@@ -328,10 +340,25 @@ setStatsText();
 function setStatsText(): void {
   $guessesTxt.textContent = guesses.toString();
   $correctTxt.textContent = numCorrect.toString();
+
   if (!(numCorrect / guesses)) {
-    $accuracyTxt.textContent = '100%';
+    $accuracyTxt.textContent = '0%';
   } else {
-    $accuracyTxt.textContent = (numCorrect / guesses).toString() + '%';
+    $accuracyTxt.textContent =
+      Math.floor((numCorrect / guesses) * 100).toString() + '%';
+  }
+
+  if (completedSquares === gridSize * gridSize && $statsModal) {
+    $statsModal.classList.remove('hidden');
+    const $statsHeading = document.querySelector(
+      '.stats-heading',
+    ) as HTMLHeadingElement;
```

Comment:
Does this exist everywhere that you could query for this outside of the function? This would be better since you'll only query once instead of every time you set the stats.